op-node/node/node.go

```diff
@@ -587,11 +587,11 @@ func (n *OpNode) Start(ctx context.Context) error {
 	}
 
 	if n.cfg.Rollup.CaffNodeConfig.IsCaffNode {
-		go func() {
-			if err := n.l2Driver.SyncDeriver.Derivation.EspressoStreamer().Start(ctx); err != nil {
```

Comment:
Espresso streamer either needs to get Start back - and gain a Mutex protecting it - or we need to drive it manually in op-node, similar to what we do in batcher.

Reply:
Personally, I'm in favor of self-driving streamer, we could also switch it to websocket API to make it faster.

Reply:
No strong preference. It looks like a self-driving streamer would require fewer changes but we need to be more careful about concurrency handling.

Reply:
Follow-up: https://app.asana.com/1/1208976916964769/project/1209392461754458/task/1209980878478930?focus=true